package.json

```diff
@@ -14,13 +14,16 @@
   "devDependencies": {
```

Comment:
I tried using chrome driver with install locally and it didn't work, it had to be installed globally. Can confirm when I come back round to setting up travis (Will be the next PR).